pygmt/src/plot.py

```diff
@@ -209,6 +209,14 @@ def plot(self, x=None, y=None, data=None, size=None, direction=None, **kwargs):
     extra_arrays = []
     if "S" in kwargs and kwargs["S"][0] in "vV" and direction is not None:
         extra_arrays.extend(direction)
+    elif (
+        "S" not in kwargs
+        and data is not None
+        and data is object
+        and "_geometry_column_name" in data.__dict__
+        and (data.geom_type == "Point").all()
+    ):
+        kwargs["S"] = "s0.2c"
```

Comment:
I'm ok with a 0.2cm square as a default style for points, but wanted to hear from @GenericMappingTools/pygmt-maintainers too. Would other [shapes](https://docs.generic-mapping-tools.org/latest/plot.html#s) be better? E.g. **p**oint (a 1 pixel dot), **c**ircle, etc? Also, is 0.2cm a good size, or should we just go without setting a size, i.e. `kwargs["S"] = "s"`.

Reply:
I would prefer either **p**oint (since no unit is required) or **c**ircle (currently not sure which size would be a good choice).